nodes/nomos-executor/src/config/mod.rs

```diff
@@ -33,6 +35,7 @@ pub struct Config {
     pub http: <ExecutorApiService as ServiceData>::Settings,
     pub cryptarchia: <crate::ExecutorCryptarchia as ServiceData>::Settings,
     pub storage: <crate::StorageService<RocksBackend<Wire>> as ServiceData>::Settings,
+    pub mempool: MempoolConfig,
```

Comment:
I did not know how else to read this config here. Taking the whole service settings basically means taking an empty tuple for the mock mempool and some libp2p settings which I guess are not important since they are not specified already right now, so I figured we would need a new type to include include the recovery path.

Reply:
Also I would like to reduce duplication and move this somewhere where it makes sense, and refer to it from both `nomos-node` and `nomos-executor`, but I would like to get opinion on whether it makes sense, and if it does, where it would go.

Reply:
The configuration is different for both. It would be nice if we could flatten one into the other though. Think of an executor as an extra behaviour on top of a `nomos-node`. Maybe it can be done with a macro :thinking: wdyt?

Reply:
Ah ok, ok. Didn't understood it. My bad. I think the `MempoolConfig` can probably go service level or in `mockpool` if its the only one that have it. Then it can be accessed from the binaries.

Reply:
This is a node-level configuration, so I personally don't think it belongs to the service level. I also think it does not belong to the mock pool itself, as it already specifies a settings type which includes both the pool and the network adapter settings, so this would only introduce confusion. If anything, I would perhaps introduce a new crate, nodes common, where we can put stuff that is common to both validator, executor, and any other node binary we might need in the future? What do you think?

Reply:
Yeah makes sense, thanks. I changed it in https://github.com/logos-co/nomos-node/pull/1049/commits/c4cf5329501bdddda1691270971700c59fb1be5b.